using-kubecost/navigating-the-kubecost-ui/savings/cluster-right-sizing-recommendations.md

```diff
@@ -33,25 +35,49 @@ Finally, you can select _Edit_ to provide information about the function of your
 
 With this information provided, Kubecost can provide the most accurate recommendations for running your clusters efficiently. By following some additional steps, you will be able to adopt Kubecost's recommendation, applying it directly to your cluster.
 
-## Adopting cluster recommendations
-
-{% hint style="warning" %}
-Adoption of cluster right-sizing recommendations is only available for clusters on GKE, EKS, or Kops-on-AWS with the Cluster Controller enabled. This is because, in order for Kubecost to apply a recommendation, it needs write access to your cluster. Write access to your cluster is enabled with the Cluster Controller.
-{% endhint %}
+## Adopting cluster right-sizing recommendations
 
 ### Prerequisites
 
-To adopt cluster right-sizing recommendations, you must first:
+To receive cluster right-sizing recommendations, you must first:
+
+* Have a GKE/EKS/AWS Kops cluster
+
+To adopt cluster right-sizing recommendations, you must:
 
 * Have a GKE/EKS/AWS Kops cluster
```

Comment:
To _receive_ recommendations we support AKS as well. Automatic adoption via Cluster Controller is limited to GKE/EKS/Kops-on-AWS